test/kitchen/site-cookbooks/dd-system-probe-check/recipes/windows.rb

```diff
@@ -18,6 +18,14 @@
   source "https://github.com/wixtoolset/wix3/releases/download/wix3112rtm/wix311-binaries.zip"
 end
 
+##
+## temporarily disable recv segment coalescing.  This appears to be
+## the root cause of the azure bug causing things to hang.
+## remove this once bug is fixed
+execute 'disable RSC' do
+  command "powershell /c disable-netadapterrsc *"
```

Comment:
Should we harmonize the recipes and use `netsh.exe int tcp set global rsc=disable` everywhere?